commerce_coordinator/apps/rollout/pipeline.py

```diff
@@ -35,32 +38,53 @@ def run_filter(self, request):  # pylint: disable=arguments-differ
         Returns:
             dict:
                 active_order_management_system: result from pipeline steps to determine if
-                            redirect_to_commercetools_checkout flag is enabled and course_run_key
-                            and sku query params exist and detect which checkout to redirect to
+                            redirect_to_commercetools_checkout flag is enabled and course_run_key,
+                            sku, and bundle query params exist and detect which checkout to redirect to
         """
-        sku = request.query_params.get('sku', '').strip()
+        sku_params = request.query_params.getlist('sku')
         course_run = request.query_params.get('course_run_key', '').strip()
-        commercetools_available_course = None
+        bundle = request.query_params.get('bundle', '').strip()
+        commercetools_available_product = None
 
-        if course_run and is_redirect_to_commercetools_enabled_for_user(request):
+        if is_user_enterprise_learner(request):
```

Comment:
Can we please add a test for this condition check?

Reply:
added